### PR TITLE
add archlinux progress 202209

### PR DESCRIPTION
**Pull request by saeziae**

### CODE REVIEW

File: roadmap/checklist.md [hunk 1]
```diff
@@ -48,68 +50,69 @@
...
+| 语言            | perl          |           | ✅ 5.36.0                      |        |        |          |                 |        |         |        | ⚠️ perl-5.34.0                                 | ✅ perl-5.30.0       |        |
+|                 | python        |           | ✅ 3.10.7                      |        |        |          | ✅ python-3.9.7 |        |         |        | ✅ python-3.10.2                               | ✅ python-3.8        |        |
+|                 | rust          |           | ✅ 1.63.0                      |        |        |          | ✅ rust-1.63.0  |        |         |        | ✅ rust-1.58.1                                 | ✅ rust-1.59.0       |        |
+|                 | ruby          |           | ✅ 3.0.4p208                   |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | openJDK       |           | ✅ 11.0.15 (openJDK & Bisheng) |        |        |          | ✅ openjdk-11   |        |         |        | ✅ openjdk-11 (bisheng & dragonwell & OpenJDK) | ✅ openjdk-8         |        |
```
Comment (Xeonacid) [inline roadmap/checklist.md]: Arch Linux RISC-V also has OpenJDK 17 and 18.

Comment (saeziae) [reply roadmap/checklist.md]: Oh I forgot that as my default jdk is 11... However openJDK 18 is not found.
``` plain
estela@starfive ~/archpkgs  sudo pacman -Ss openjdk
extra/jdk-openjdk 17.0.1.u12-1 [installed]
    OpenJDK Java 17 development kit
extra/jdk11-openjdk 11.0.15.u10-1 [installed]
    OpenJDK Java 11 development kit
extra/jdk17-openjdk 17.0.4.1.u1-2
    OpenJDK Java 17 development kit
```

Comment (Xeonacid) [reply roadmap/checklist.md]: My fault, I misremembered. :(

File: roadmap/checklist.md [hunk 2]
```diff
@@ -48,68 +50,69 @@
...
+|                 | ruby          |           | ✅ 3.0.4p208                           |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | openJDK       |           | ✅ 17.0.1, 11.0.15 (openJDK & Bisheng) |        |        |          | ✅ openjdk-11   |        |         |        | ✅ openjdk-11 (bisheng & dragonwell & OpenJDK) | ✅ openjdk-8         |        |
+|                 | Nodejs        |           | ✅ 18.9.1                              |        |        |          | ⚠️              |        |         |        | ✅ nodejs-v16.15.1                             | ✅ nodejs-12.22.9    |        |
+|                 | Ocaml         |           |                                        |        |        |          |                 |        |         |        | ✅ ocaml-4.12.0-3                              |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/ocaml 4.14.0-1

File: roadmap/checklist.md [hunk 3]
```diff
@@ -48,68 +50,69 @@
...
+|                 | golang        |           | ✅ 1.19                                |        |        |          | ✅ golang-1.19  |        |         |        | ✅ golang-1.18.3                               |                      |        |
+|                 | erlang        |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: community/erlang 25.0.4-1

File: roadmap/checklist.md [hunk 4]
```diff
@@ -48,68 +50,69 @@
...
+| 构建工具        | obs           |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | osc           |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | obs-server    |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+| 容器            | runc          |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: community/runc 1.1.4-1

File: roadmap/checklist.md [hunk 5]
```diff
@@ -48,68 +50,69 @@
...
+|                 | docker        |           | ✅ 20.10.17                            |        |        |          |                 |        |         |        |                                                | ✅ docker.io-19.03.8 |        |
+| 桌面            | Xfce Desktop  |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/xfce4-session 4.16.0-2

File: roadmap/checklist.md [hunk 6]
```diff
@@ -48,68 +50,69 @@
...
+|                 | KDE           |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/plasma-workspace 5.25.5-1
extra/plasma-framework 5.98.0-1

File: roadmap/checklist.md [hunk 7]
```diff
@@ -48,68 +50,69 @@
...
+|                 | LXDE          |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | GNOME         |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | Deepin        |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | MATE          |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: community/mate-session-manager 1.26.0-1

File: roadmap/checklist.md [hunk 8]
```diff
@@ -48,68 +50,69 @@
...
+|                 | LXQT          |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: community/lxqt-session 1.1.1-1

File: roadmap/checklist.md [hunk 9]
```diff
@@ -48,68 +50,69 @@
...
+|                 | Enlightenment |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: community/enlightenment 0.25.4

File: roadmap/checklist.md [hunk 10]
```diff
@@ -48,68 +50,69 @@
...
+| 浏览器          | firefox       |           |                                        |        |        |          |                 |        |         |        | ✅                                             |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/firefox 105.0-2

File: roadmap/checklist.md [hunk 11]
```diff
@@ -48,68 +50,69 @@
...
+|                 | chromium      |           |                                        |        |        |          |                 |        |         |        | ✅                                             |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/chromium 104.0.5112.101-1

File: roadmap/checklist.md [hunk 12]
```diff
@@ -48,68 +50,69 @@
...
+| 图像浏览编辑    | GIMP          |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/gimp 2.10.32-2

File: roadmap/checklist.md [hunk 13]
```diff
@@ -48,68 +50,69 @@
...
+| 多媒体播放器    | VLC           |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/vlc 3.0.17.4-8

File: roadmap/checklist.md [hunk 14]
```diff
@@ -48,68 +50,69 @@
...
+| 办公软件        | LibreOffice   |           |                                        |        |        |          |                 |        |         |        | ✅                                             |                      |        |
+|                 | Thunderbird   |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/thunderbird 91.12.0-1

File: roadmap/checklist.md [hunk 15]
```diff
@@ -48,68 +50,69 @@
...
+|                 | PyCharm       |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+| DB              | sqlite        |           | ✅ 3.39.1                              |
+|                 |               |           |                                        |        |        |          | ✅              |        |         |
+|                 | Mysql         |           | mariadb-10.9.3                         |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | postgresql    |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/postgresql 14.5-1

File: roadmap/checklist.md [hunk 16]
```diff
@@ -48,68 +50,69 @@
...
+| http/web 服务器 | Apache2       |           | ✅ 2.4.54                              |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | Nginx         |           | ✅ 1.22.0                              |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | openssl       |           | ✅ 1.1.1.q                             |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | HAProxy       |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: community/haproxy 2.6.6-1

File: roadmap/checklist.md [hunk 17]
```diff
@@ -48,68 +50,69 @@
...
+|                 | Lighttpd      |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/lighttpd 1.4.67-1

File: roadmap/checklist.md [hunk 18]
```diff
@@ -48,68 +50,69 @@
...
+|                 | Tengine       |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | Squid         |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: community/squid 5.7-1

File: roadmap/checklist.md [hunk 19]
```diff
@@ -48,68 +50,69 @@
...
+|                 | Varnish       |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/varnish 7.1.1-1

File: roadmap/checklist.md [hunk 20]
```diff
@@ -48,68 +50,69 @@
...
+| web             | flask         |           | ✅                                     |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: community/python-flask 2.2.2-1

File: roadmap/checklist.md [hunk 21]
```diff
@@ -48,68 +50,69 @@
...
+| 大数据          | HBase         |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | Hadoop        |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | hive          |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+| 中间件          | Redis         |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: community/redis 7.0.5-1

File: roadmap/checklist.md [hunk 22]
```diff
@@ -48,68 +50,69 @@
...
+|                 | MemCache      |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/memcached 1.6.10-1

File: roadmap/checklist.md [hunk 23]
```diff
@@ -48,68 +50,69 @@
...
+|                 | mybatis       |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | Dubbo         |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | zookeeper     |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
+|                 | numpy         |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: extra/python-numpy 1.23.3-1

File: roadmap/checklist.md [hunk 24]
```diff
@@ -48,68 +50,69 @@
...
+|                 | scipy         |           |                                        |        |        |          |                 |        |         |        |                                                |                      |        |
```
Comment (felixonmars) [inline roadmap/checklist.md]: community/python-scipy 1.9.1-1

**Review by felixonmars — CHANGES_REQUESTED** [review]

**Pull request merged**